sql/src/main/java/com/vmware/ddlog/DDlogJooqProvider.java

```diff
@@ -22,21 +25,27 @@
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Locale;
 import java.util.Map;
 
+import static org.jooq.impl.DSL.field;
+
 public class DDlogJooqProvider implements MockDataProvider {
     private static final String INTEGER_TYPE = "java.lang.Integer";
     private static final String STRING_TYPE = "java.lang.String";
     private static final String BOOLEAN_TYPE = "java.lang.Boolean";
     private static final String LONG_TYPE = "java.lang.Long";
-
+    private static final String DDLOG_SOME_TYPE = "ddlog_std::Some";
```

Comment:
technically these are two different constructors of the same type, the option type.

Reply:
So can I use just use `ddlog_std::Option` instead?

Reply:
Your code is correct, it's just that these two are constructors, not types.